nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -544,17 +613,18 @@ public void skip_field_nullable_explicit_nonnull() {
             "   // BUG: Diagnostic contains: field f not initialized",
             "   @Nonnull Object f;",
             "}")
-        .expectNoFixSerialization()
+        .expectNoOutput()
+        .setFactory(fixDisplayFactory)
+        .setOutputFileNameAndHeader(SUGGEST_FIX_FILE_NAME, SUGGEST_FIX_FILE_HEADER)
         .doTest();
   }
 
   @Test
-  public void test_custom_annot() {
+  public void SuggestCustomAnnotTest() {
```

Comment:
This is a method, name should start with lowercase.

Reply:
ac2ec384b9d7f2a0d391696bc2edc08904d386d8